flycheck.el

```diff
@@ -6933,8 +6940,17 @@ See URL `https://www.rebar3.org'."
    (error line-start
           (file-name) ":" line ": " (message) line-end))
   :modes erlang-mode
+  :enabled (lambda () (flycheck-rebar-config-exists-p))
+  :predicate (lambda () (flycheck-buffer-saved-p))
   :working-directory flycheck-rebar3-project-root)
 
+(defun flycheck-rebar-config-exists-p ()
+  "Whether there is a rebar.config for the current buffer."
+  (let* ((executable (flycheck-find-checker-executable 'erlang-rebar3))
```

Comment:
Is there a reason why you check that?  Are you aware that Flycheck does that automatically?